src/showcase/src/pages/[page].astro

```diff
@@ -0,0 +1,21 @@
+---
+import "$src/styles/main.css";
+import { iiPages } from "../../../showcase/src/showcase";
+
+export function getStaticPaths() {
+  return Object.keys(iiPages).map((key) => ({ params: { page: key } }));
+}
+
+const { page } = Astro.params;
+---
+
+<div id="pageContent" data-page-name={page}></div>
+<script>
+  import { iiPages } from "../../../showcase/src/showcase";
+
+  const page = (document.querySelector("[data-page-name]") as HTMLDivElement)
+    .dataset?.pageName as string;
```

Comment:
Any particular reason not using `define:vars` to access the `page` variable?

Reply:
Nice! Looked for something like this but couldn't find it

Reply:
Actually, this then inlines the script which creates other issues. I'll have a look at this separately but will leave as is for now.